modules/aws-emr-ports/output.tf

```diff
@@ -0,0 +1,21 @@
+locals {
+  applications  = [for app in var.applications : lower(app)]
+  running_hbase = contains(local.applications, "hbase")
+  running_spark = contains(local.applications, "spark")
+  output_hbase  = concat(var.master_ports_emr, var.master_ports_hbase, var.master_ports_ganglia, var.additional_ports)
+  output_spark  = concat(var.master_ports_emr, var.master_ports_spark, var.additional_ports)
```

Comment:
let's be explicit in these variable names on whether they refer to the master or core ports.

Reply:
Done